src/deps.txt

```diff
@@ -49,3 +49,9 @@ ignition
 
 # for grub install when creating images without anaconda
 grub2-pc
+
+# Mantle suite
+#FEDORA mantle-kola
```

Comment:
This isn't Fedora specific.

Reply:
the rpm doesn't exist in EL right now so I was thinking it would be better to not include it?

I guess other changes I've made make it where we don't support both ways anyway so i should just force it on both and require us to get the rpm in.

Reply:
sound good?

Reply:
It'd totally break RHCOS to drop this though.

Reply:
(And yeah, the additional RPM requirement is another big pain point sadly...)